### PR TITLE
Updates from PWA Studio docs repo

### PR DESCRIPTION
**Pull request by bdenham**
# Description

Docs content for PWA Studio Release 12.0.0 and 12.1.0. The changes are mostly from the auto-generated docs from pwa-studio repo.

The remaining additions that I will add in the next PRs are:

1. Port over the commerce feature support table.
2. Add new metapackages section with Readme text from the metapackage repos.
3. Add new GraphQL docs for Contact Us and Newsletter to metapackages section

## Related Issue

Jira Ticket: [PWA-2376](https://jira.corp.magento.com/browse/PWA-2376)

## How Has This Been Tested?

Built gh-pages site to review the changes: https://developer-stage.adobe.com/commerce/pwa-studio/integrations/adobe-commerce/version-compatibility/

### CODE REVIEW

**Comment by bdenham** [comment]
@jcalcaben, if you can spare some time this morning to review this, that would be most helpful. Most changes are from the auto-generated files I copied from the most recent build in pwa-studio. Thanks.

**Comment by bdenham** [comment]
@jcalcaben Just pushed the Venia sample data changes from [PWA-2151/3473](https://github.com/magento/pwa-studio/pull/3473/files)

File: src/pages/guides/packages/venia/sample-data/index.md [hunk 1]
```diff
@@ -1,54 +1,64 @@
...
+```
+
+## Step 2. Require in the sample data
 
-Execute the script in the root directory of your application to add the Venia sample data modules to the backend application:
+Run the following command in the Magento root directory to update the composer.json to require in magento/venia-sample-data metapackage:
 
 ```sh
-bash deployVeniaSampleData.sh
+composer require --no-interaction --ansi magento/venia-sample-data:0.0.1
```
Comment (jcalcaben) [inline src/pages/guides/packages/venia/sample-data/index.md:37]: would be good to mention that the version should be the latest released and not always be `0.0.1`

File: src/pages/api/venia/components/general/Link/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: Link;
+---
+
+import Link from '/src/data/auto-generated/venia-ui/lib/components/Link/link.md';
+
+# Link
+
+The `Link` component wraps the [react-router-dom Link](https://knowbody.github.io/react-router-docs/api/Link.html) component, but adds intelligent prefetching behavior to speed up performance. Our `Link` component binds to an `IntersectionObserver` to prefetch the target page only when the link is visible in the viewport. And when possible, our `Link` loads the page from local cache to prevent unnecessary roundtrips to the server. In short, the `Link` component builds in all the best-practices that Google recommends when prefetching content for fast transitions.
```
Comment (jcalcaben) [inline src/pages/api/venia/components/general/Link/index.md:9]: this links to an [unofficial docs site](https://knowbody.github.io/react-router-docs/). the official docs site for react router is: https://v5.reactrouter.com/web/api/Link

**Review by jcalcaben — CHANGES_REQUESTED** [review]

**Review by jcalcaben — APPROVED** [review]

**Pull request merged**